libgstd/Makefile.am

```diff
@@ -71,3 +73,7 @@ libgstd_@GSTD_API_VERSION@_la_CFLAGS =                                   \
 			 $(JANSSON_CFLAGS) -pthread
 
 libgstd_@GSTD_API_VERSION@_la_LDFLAGS = $(GST_LIBS) $(GIO_LIBS) $(GJSON_LIBS) $(LIBSOUP_LIBS) $(JANSSON_LIBS) -pthread
+
+noinst_HEADERS = \
```

Comment:
Are libgstd_json.h and libgstd_parser.h public header files?
If those files are public you need to install the files